crates/sncast/tests/e2e/script/deploy.rs

```diff
@@ -60,6 +60,8 @@ async fn test_same_salt_and_class_hash_deployed_twice() {
     assert_stdout_contains(
         output,
         indoc! {r#"
+        [..]Compiling[..]
```

Comment:
nit: is it possible to get rid of this? I thought `assert_stdout_contains` can be used to check if output contains some sequence, so we get to choose what we want to match. scarb output is something we do not need to assert in our tests

Reply:
yeah, maybe. 
how come it did not complain earlier, do you happen to know?

Reply:
Good question actually since the output inheriting from Scarb didn't change 🤔